src/components/spatial/Spatial.js

```diff
@@ -33,6 +107,7 @@ export default class Spatial extends AbstractSelectableComponent {
     this.cellsData = [];
     this.neighborhoodsData = [];
     this.images = [];
+    this.tiff = [];
```

Comment:
Do you think tiff and images will coexist? if we could just replace it, and say there is one right way to do images with Vitessce, I'd be happy.

Reply:
I'm also confused: I had though for your work, tiff and zarr were alternatives? Why are there references to both in this PR?

Reply:
We should talk to Nils about tiff + images.  I'm calling this `tiff` because that is where I anticipate us going.  `data` or something like it seemed too generic and anything else seemed wordy.

Reply:
maybe `raster` or `rasters`?

Reply:
Perfect...I was too fixated on the word "microscopy"